packages/dds/tree/src/feature-libraries/flex-tree/index.ts

```diff
@@ -61,6 +61,6 @@ export {
 	FlexibleNodeSubSequence,
 } from "./flexTreeTypes.js";
 
-export { NodeKeys } from "./nodeKeys.js";
+export { NodeKeys } from "../../simple-tree/nodeKeys.js";
```

Comment:
this should be removed: flex tree should not depend on simple-tree.

Reply:
Removed unnecessary export